From 476a15168d26edff26c794a3d6b14ef77b68b7bb Mon Sep 17 00:00:00 2001
From: rikardo <rikard.olander@hostmobility.com>
Date: Wed, 9 Jan 2019 14:59:04 +0100
Subject: [PATCH] Add-Hm-specific-device-tree-to-makefile

---
 arch/arm/dts/Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 69bee95..0dcb0e1 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -49,10 +49,11 @@ dtb-$(CONFIG_TEGRA) += tegra20-harmony.dtb \
 	tegra20-ventana.dtb \
 	tegra20-whistler.dtb \
 	tegra20-colibri.dtb \
+	tegra20-mx4.dtb \
+	tegra30-mx4.dtb \
 	tegra30-apalis.dtb \
 	tegra30-beaver.dtb \
 	tegra30-cardhu.dtb \
-	tegra30-colibri.dtb \
 	tegra30-tec-ng.dtb \
 	tegra114-dalmore.dtb \
 	tegra124-apalis.dtb \
@@ -279,6 +280,8 @@ dtb-$(CONFIG_MACH_SUN9I) += \
 
 dtb-$(CONFIG_VF610) += vf500-colibri.dtb \
 	vf610-colibri.dtb \
+	vf610-mx4-c61.dtb \
+	vf610-mx4-v61.dtb \
 	vf610-twr.dtb \
 	pcm052.dtb \
 	bk4r1.dtb
-- 
2.7.4

